Website/src/rendering/src/assets/css/components/Speakers/SpeakersGrid.css

```diff
@@ -27,7 +29,7 @@
       }
 
       .speaker__name {
-        @apply mx-auto font-bold my-1;
+        @apply font-bold my-1;
       }
 
       .speaker__jobtitle {
```

Comment:
The `speaker__jobtitle` CSS class was removed from the element in `SpeakersGrid.tsx`. It can be removed from this CSS file as well as it is not used anymore.